### DISCUSSION TITLE
Make submitted scores count towards play time or make them not count towards play count

### DISCUSSION THREAD

**Discussion by StayBlue** [ppy/osu#16646] [Ideas] [answered]
This has probably been suggested before, but if it has, I haven't found a discussion for it so I thought I should make one.

It kind of seems counterintuitive that even scores submitted on Lazer, even on ranked maps, count towards your play count but not your playtime. This is probably something that's intentional until scores submit for PP, but it would be nice if submitted scores on Lazer counted towards your playtime or didn't count towards your play count at all.

**Comment by mk56-spn** [comment]
why does it matter if they only count towards one for now? especially considering neither metric are good indications of anything

**Reply by StayBlue** [reply]
It's just one of those things that make you scratch your head wondering as to why it doesn't count towards your playtime in the first place.

**Comment by peppy** [comment, marked as answer]
They will count to both, retroactively. There's no need for anything to change here. Play time has not yet been turned on for infrastructure reasons.
